updater/lib/dependabot/updater/operations/create_security_update_pull_request.rb

```diff
@@ -190,6 +219,7 @@ def check_and_create_pull_request(dependency)
           record_warning_notices(notices) if notices.any?
 
           create_pull_request(dependency_change)
+          true
         rescue Dependabot::AllVersionsIgnored
```

Comment:
**Review Tip:** Using boolean to understand if update process (succeed, fail, no update)